admin_guide/cluster_management.rst

```diff
@@ -42,6 +42,31 @@ The Citus master only stores metadata about the table shards and does not store
 
 However, in some write heavy use cases where the master becomes a performance bottleneck, users can add another master. As the metadata tables are small (typically a few MBs in size), it is possible to copy over the metadata onto another node and sync it regularly. Once this is done, users can send their queries to any master and scale out performance. If your setup requires you to use multiple masters, please `contact us <https://www.citusdata.com/about/contact_us>`_.
 
+Multi-tenancy
+#############
+
+Citus provides ways to customize table distribution for the multi-tenant use case. By default Citus assigns table rows to shards using the table's distribution column and method. Many multi-tenant SaaS providers need to override this logic for particular tenants for a few reasons:
```

Comment:
This paragraph is a bit different from Metin's original documentation of this feature. Primarily, it doesn't override any table-distribution logic, or allow customization of this logic. It still uses hash-partitioning, but just isolates the hash-value for the provided tenant into its own shard. Given this, Citus still assigns rows to shards using the distribution column and method.